plugin/trino-kinesis/src/main/java/io/trino/plugin/kinesis/KinesisStreamFieldGroup.java

```diff
@@ -42,6 +47,12 @@ public String getDataFormat()
         return dataFormat;
     }
 
+    @JsonProperty
```

Comment:
I am not sure if this going to work. You would need to add round trip json serialization test to verify this. Maybe simply remove this annotation?

Reply:
I've just remove the annotation